UCLA Radio/UCLA Radio/NotificationManager.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  NotificationManager.swift
+//  UCLA Radio
+//
+//  Created by Nathan Smith on 12/6/16.
+//  Copyright © 2016 ChrisLaganiere. All rights reserved.
+//
+
+import Foundation
+import UserNotifications
+
+class NotificationManager {
+    
+    static let sharedInstance = NotificationManager()
+    
+    /*
+    var notificationsAuthorized:Bool {
+        get {
+            if #available(iOS 10.0, *) {
+                let center = UNUserNotificationCenter.current()
+                center.getNotificationSettings() {_ in 
+                    
+                }
+            }
+        }
+    }*/
+    
+    func requestNotificationPermission(application: UIApplication) {
+        // Configure Notifications
+        // User UNUSerNotificationCenter if >= iOS 10, UIUserNotificationSettings otherwise
+        if #available(iOS 10.0, *) {
+            let center = UNUserNotificationCenter.current()
+            center.requestAuthorization(options: [.alert, .badge, .sound]) { (granted, error) in
+                if !granted {
+                    print("Error: Notifications not authorized by user.")
+                }
+            }
+        } else {
+            let settings = UIUserNotificationSettings(types: [.alert, .badge, .sound], categories: nil)
+            application.registerUserNotificationSettings(settings)
+        }
+    }
+    
+    func notificationsToggledForShow(_ show: Show, isOn: Bool) {
```

Comment:
more appropriate to call it something with an action like `toggleNotificationsForShow:`